api/envoy/api/v2/auth/cert.proto

```diff
@@ -118,9 +118,9 @@ message PrivateKeyProvider {
 
   // Private key method provider specific configuration.
   oneof config_type {
-    google.protobuf.Struct config = 2 [deprecated = true];
+    google.protobuf.Struct config = 2 [deprecated = true, (udpa.annotations.sensitive) = true];
```

Comment:
Doesn't this effectively not do anything since redaction doesn't work inside the struct?

Reply:
No, this works fine: it recursively redacts everything inside the struct.
The case that wouldn't work is if the struct _weren't_ annotated as `sensitive`, but contained some field that, if it were part of a strongly-typed message, would have been annotated `sensitive`. That is, when we're looking inside the struct, we have no info about which fields should be redacted.

Reply:
OK got it. Since this is a struct that gets converted into a strongly typed message everything works. :+1: